### ISSUE TITLE
Main parameter array

### ISSUE THREAD

**Issue by simonrobert1** [armin-pfaeffle/sass-autocompile#43]
Hi,
It would be great to be able to use an array as the main parameter. 
Ex : 
//main: ["../stylesheet.scss","../print.scss"]

Thank you,

Simon

**Comment by armin-pfaeffle** [comment]
Hey! Well, that's not that big problem to integrate, but what is the expected behaviour? Do you need that for coding a framework, so you do not have to compile the defalt and the print stylesheet separatly?

**Comment by simonrobert1** [comment]
I should trigger the compilation of each Sass file in the array. Some partials need to be in both stylesheet.css and print.css.

**Comment by smnbbrv** [comment]
@armin-pfaeffle I fully agree with @simonrobert1 , I realized I need it as well. I don't think it's hard to do, so if you have no time to do that I can try my best and make a pull request. What do you think?

**Comment by nahuelhds** [comment]
I have a similar situation than @simonrobert1. An array solution for the main parameter would be very helpful (I've mention this in every ticket, lol).

I'm more on this approach, though
`//main: ../stylesheet.scss, ../print.scss`

But of course it's just the same if the value is an actual array.

Thanks!

**Comment by davidchoy** [comment]
This would be super useful! In my case, I'm creating similar, but different, front-end and back-end style sheets for the Wordpress and its Gutenberg editor. Auto compiling to style.css and style-editor.css would be really useful.

**Comment by macs7** [comment]
Hello, any news about this feature? Do you know if an Atom package could manage this? Thanks
